File: src/main/java/com/xudong/vam/model/SelectPackage.java
package com.xudong.vam.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.NonNull;
import lombok.RequiredArgsConstructor;

@Entity
@Data
@RequiredArgsConstructor
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "t_select_package")
public class SelectPackage {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @NonNull
    @Column(name = "name", length = 128)
    private String name;

    @NonNull
    @Column(name = "uuid")
    private String uuid;
}

File: src/main/java/com/xudong/vam/core/model/SelectPackage.kt
package com.xudong.vam.core.model

import jakarta.persistence.*

@Entity
@Table(name = "t_select_package")
data class SelectPackage(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,

    @Column(name = "name", length = 128)
    var name: String = "",

    @Column(name = "uuid")
    var uuid: String = "",
)